timer.py

```diff
@@ -0,0 +1,30 @@
+import time 
+
+def printTime(func):
```

Comment:
According to [PEP-8](https://peps.python.org/pep-0008/#descriptive-naming-styles) function names should be lowercase with underscores, so `print_time()` in this case.